testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/WildflyHelmChartExistingValuesOpenShiftExampleApplication.java

```diff
@@ -55,20 +54,17 @@ public Map<String, String> getSetOverrides() {
 		return setOverrides;
 	}
 
-	private HelmWildflyRelease loadRelease() {
-		URL url = this.getClass().getResource("wildfly-helm-values.yaml");
-		if (url == null) {
-			throw new IllegalStateException("No wildfly-helm-values.yaml found");
-		}
-		try {
-			Path valuePath = Path.of(url.toURI());
-			ObjectMapper mapper = new ObjectMapper(new YAMLFactory().disable(YAMLGenerator.Feature.WRITE_DOC_START_MARKER));
-			return mapper.readValue(valuePath.toFile(), HelmWildflyRelease.class);
-		} catch (Error | RuntimeException e) {
-			throw e;
-		} catch (Exception e) {
-			throw new RuntimeException(e);
-		}
+	private HelmChartRelease loadRelease() {
+		if (IntersmashTestsuiteProperties.isCommunityTestExecutionProfileEnabled()) {
+			return new WildFlyHelmChartReleaseAdapter(
+					HelmChartReleaseAdapter.<HelmWildflyRelease> fromValuesFile(
+							this.getClass().getResource("wildfly-helm-values.yaml"), HelmWildflyRelease.class));
+		} else if (IntersmashTestsuiteProperties.isProductizedTestExecutionProfileEnabled()) {
+			return new Eap8HelmChartReleaseAdapter(
+					HelmChartReleaseAdapter.<HelmEap8Release> fromValuesFile(
+							this.getClass().getResource("eap8-helm-values.yaml"), HelmEap8Release.class));
```

Comment:
@fabiobrz since neither "wildfly-helm-values.yaml" nor "eap8-helm-values.yaml" define builder and runtime images (which are set in WildflyHelmChartOpenShiftExampleApplication.loadRelease) I suspect we should add such a setting to WildflyHelmChartExistingValuesOpenShiftExampleApplication.loadRelease in order to use the actual images passed to the test run, e.g.:

```java
	private HelmChartRelease loadRelease() {
		if (IntersmashTestsuiteProperties.isCommunityTestExecutionProfileEnabled()) {
			WildFlyHelmChartReleaseAdapter wildFlyHelmChartReleaseAdapter = return new WildFlyHelmChartReleaseAdapter(
					HelmChartReleaseAdapter.<HelmWildflyRelease> fromValuesFile(
							this.getClass().getResource("wildfly-helm-values.yaml"), HelmWildflyRelease.class));
			wildFlyHelmChartReleaseAdapter.setJdk17BuilderImage(...);
			wildFlyHelmChartReleaseAdapter.setJdk17RuntimeImage(...);
			return wildFlyHelmChartReleaseAdapter;
		} else if (IntersmashTestsuiteProperties.isProductizedTestExecutionProfileEnabled()) {
			Eap8HelmChartReleaseAdapter eap8HelmChartReleaseAdapter = new Eap8HelmChartReleaseAdapter(
					HelmChartReleaseAdapter.<HelmEap8Release> fromValuesFile(
							this.getClass().getResource("eap8-helm-values.yaml"), HelmEap8Release.class));
			eap8HelmChartReleaseAdapter.setJdk17BuilderImage(...);
			eap8HelmChartReleaseAdapter.setJdk17RuntimeImage(...);
			return eap8HelmChartReleaseAdapter;
		} else
			throw new IllegalStateException("Not a valid testing profile!");
	}
```

Reply:
+1 fixing

Reply:
great!